src/Orchard.Web/Modules/Orchard.Users/Drivers/UserApprovePartDriver.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Laser.Orchard.StartupConfig.ApproveUserExtension.Services;
```

Comment:
Not yet a Laser project

Reply:
Sorry I was wrong, I made changes to the files.